pdns/dnsdist-protobuf.cc

```diff
@@ -33,7 +34,7 @@ static void protobufFillMessage(PBDNSMessage& message, const DNSQuestion& dq)
   message.set_inbytes(dq.len);
 
   struct timespec ts;
-  clock_gettime(CLOCK_REALTIME, &ts);
+  gettime(&ts);
```

Comment:
That's probably the only place were using CLOCK_MONOTONIC or CLOCK_MONOTONIC_RAW would be an issue. Perhaps we could add a `needRealTime` parameter to `gettime()`?